app/views/roles/edit.html.erb

```diff
@@ -17,7 +17,7 @@
       <div class="row">
         <div class="col-xs-8">
           <%= bootstrap_form_tag url: role_management.role_users_path(@role) do |f| %>
-            <%= f.collection_select('user_key', (User.all - @role.users ), :user_key, :display_name,{label: t('role-management.edit.user'), class: "form-control"}) %>
+            <%= f.collection_select('user_key', (User.all - @role.users ), :user_key, :display_name,{include_blank:'Please select a user…', label: t('role-management.edit.user'), class: "form-control"}) %>
```

Comment:
What happens when you submit this form without selecting a value?  Do you get some kind of couldn't find a user error?

You might want to look into the :disabled option as well as the :include_blank option.